playbook/app/pb_kits/playbook/pb_image/_image.scss

```diff
@@ -11,6 +11,7 @@ $image-sizes: (
 
 [class^=pb_image_kit] {
   position: relative;
+  max-width: -webkit-fill-available;
```

Comment:
If this is something that we need for the aspect ratio, then we should also polyfill for Firefox at a bare minimum as well. How does this behave in the current version of that browser?

Reply:
Im sorry, what language are you speaking Stephen?

Reply:
lol, we have no clue what polyfill means, but we will do some research and get back to you

Reply:
@thestephenmarshall so firefox 100% needs a polyfill, not necessarily for the image kit because that is working, but noticing that card kit as well as nav bar are all over the place with bugs

Reply:
<img width="1416" alt="Screen Shot 2020-11-17 at 3 19 49 PM" src="https://user-images.githubusercontent.com/64423775/99443114-59db4c80-28e8-11eb-8ef2-9fd01e436137.png">

this is what it currently looks like in Firefox